src/Cognito.re

```diff
@@ -201,6 +206,158 @@ let signUp =
           | _ => `CognitoUnknownError(msg)
           };
         Future.make(resolve => resolve(Belt.Result.Error(err)));
-      }
-    );
+      };
+    });
+};
+type confirmSignUpErrors = [ | `CognitoConfirmationCodeValidation(string)];
```

Comment:
There's a lot more of these we need to handle, see https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_ConfirmSignUp.html.

Reply:
Guess you found it. Just made a note above.

Reply:
Ahh, I actually didn't make the connection.  This is good down here, remove it from the type above. :+1: